std_msgs/String type
geometry_msgs/Point start_point
geometry_msgs/Point end_point
std_msgs/Float32 v_max
std_msgs/Float32 a_max



